modules/network-observability-operator-install.adoc

```diff
@@ -25,24 +30,17 @@ This documentation assumes that your `LokiStack` instance name is `loki`. Using
 . Choose  *Network Observability Operator* from the list of available Operators in the *OperatorHub*, and click *Install*.
 . Select the checkbox `Enable Operator recommended cluster monitoring on this Namespace`.
 . Navigate to *Operators* -> *Installed Operators*. Under Provided APIs for Network Observability, select the *Flow Collector* link.
-.. Navigate to the *Flow Collector* tab, and click *Create FlowCollector*. Make the following selections in the form view:
-+
-* *spec.agent.ebpf.Sampling* : Specify a sampling size for flows. Lower sampling sizes will have higher impact on resource utilization. For more information, see the `FlowCollector` API reference, under spec.agent.ebpf.
-* *spec.deploymentModel*: If you are using Kafka, verify Kafka is selected.
-* *spec.exporters*: If you are using Kafka, you can optionally send network flows to Kafka, so that they can be consumed by any processor or storage that supports Kafka input, such as Splunk, Elasticsearch, or Fluentd. To do this, set the following specifications:
```

Comment:
@jotak @jpinsonneau @msherif1234 @stleerh 
I was talking to @Amoghrd about this, and we were thinking why do we need to document Kafka exporter config here? We already link to it at the end of this Installing topic. especially with IPFIX now too, these steps seem outdated. Is the mention and the link to the Exporting enriched network flows topic OK? Additionally, I moved the Installing Kafka topic after the Network Observability install topic one and put a link to the Configuring the FlowCollector with Kafka topic.

Reply:
If the intention is to just get Network Observability up and running using Loki, then this would be sufficient.  However, the reality is you have to decide upfront all of the features that you want because each one of these features require you to reconfigure FlowCollector, which is disruptive because the FlowCollector pods have to be terminated and recreated again.  This includes features such as:  
  
1\. Use of Kafka  
2\. Export to Kafka consumer  
3\. Export to IPFIX collector  
4\. Conversation tracking  
5\. Support for SR-IOV interfaces in netdevice mode  
6\. Packet drops  
7\. DNS tracking  
  
Round Trip Time (RTT) will also be included once it reaches GA.  The user should consider these key aspects when they create a FlowCollector instance, including whether they will be using Loki or not.

Reply:
Doing this would avoid cluttering the Operator Install with choices while still letting customers know about the risks of restarting the FlowCollector when re-configuring and also presenting them with the choices they need to make, which are documented either in Configuring or Observing areas already.